reactDashboard/src/pages/Regional.jsx

```diff
@@ -146,7 +146,7 @@ const Regional = ({
             />
           </Col>
         </Row>
-        <Row className="d-none d-sm-flex">
+        <Row>
```

Comment:
There are now 2 hrs under the data definitions in mobile view - can you get rid of one of them?

Reply:
No longer displays extra hr on smaller devices